spatialos-sdk/examples/project-example/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Generated bindings to SpatialOS components, shouldn't be committed.
+generated_code.rs
```

Comment:
I think that's a fair point, though I think it would make more sense to have small, controlled examples as part of the spatialos-sdk-code-generator crate, rather than using the example project for that purpose. That said, I'm happy to restore `generated_code.rs` if you'd prefer 🙂